package.json

```diff
@@ -89,6 +89,14 @@
         "Right"
       ],
       "default": "Right"
+    },
+    "messageTypesToIgnoreInPanel": {
+      "description": "If you want to see a message type higlighted in editor but not in panel",
```

Comment:
"Comma separated list of message types to show _only_ in the editor, and not in the panel."
(Not sure if that italicization will show in the settings view, cut it out if not.)